content/intro-to-anchor.md

```diff
@@ -0,0 +1,511 @@
+# Anchor Program Structure
+
+# Lesson Objectives
+
+*By the end of this lesson, you will be able to:*
+
+- Use the Anchor framework to build a basic program
+- Describe the basic structure of an Anchor program
+- Explain how to implement basic account validation and security checks with Anchor
+
+# TL;DR
+
+- **Anchor** is a framework building Solana programs
+- **Anchor** macros speed up the process of building Solana programs by abstracting away a significant amount of boilerplate code
+- **Anchor** allows you more easily build secure programs by performing certain security checks, requiring account validation, and providing a simply way to implement additional checks.
```

Comment:
From -> **Anchor** allows you more easily
To: **Anchor** allows you to more easily